test_runner/pg_clients/rust/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM rust:1.66 AS build
+WORKDIR /source
+COPY . .
+RUN cargo build --release
+
+FROM rust:1.66
```

Comment:
you can derive from some minimalistic image without rust at all. For instance, compute uses `FROM debian:bullseye-slim`